### DISCUSSION TITLE
DevOps Practices for Continuous Deployment with Batect

### DISCUSSION THREAD

**Discussion by lmortimer** [batect/batect#889] [Show and tell]
I use Batect at work and absolutely love the tool. We practice Trunk Based Development and Continuous Deployment with a 15 minute lead time. Batect is **the** key enabler for our way of working. I've written a blog on how we use Batect and the advantages it brings, specifically around:

- Development Environments - Reproducible, immutable, and as close to production as possible.
- Optimising for Developer Experience - Make the day to day tasks we run easy.
- Simplifying the build pipeline - Whatever runs in CI can be run locally

Have a read: [DevOps Practices for Continuous Deployment](https://isthisit.nz/posts/2021/devops-practices-for-continuous-deployment/)

Happy to answer any questions here. Cheers 🚀

**Comment by charleskorn** [comment]
This is really cool, thanks for sharing your experiences @lmortimer, and glad to hear you're finding Batect useful!
